filebeat/fileset/modules.go

```diff
@@ -115,7 +117,7 @@ func newModuleRegistry(modulesPath string,
 }
 
 // NewModuleRegistry reads and loads the configured module into the registry.
-func NewModuleRegistry(moduleConfigs []*common.Config, beatVersion string, init bool) (*ModuleRegistry, error) {
+func NewModuleRegistry(moduleConfigs []*common.Config, beatInfo beat.Info, init bool) (*ModuleRegistry, error) {
```

Comment:
Why are you passing `beat.Info` when only the `Version` attribute is used in the `ModuleRegistry`?